tools/README.md

```diff
@@ -0,0 +1,4 @@
+## software documentation
+"Portapack_Hardware_version_select.exe" Build Based on .Net Framework 4.6 
+
+1,This software is used to set the hardware version!
```

Comment:
Hi @zhang00963  Do you by any chance have the source data available.
That would make it possible to port it to other platforms.

Reply:
@zhang00963  do you have source data available?

Reply:
@ArjanOnwezen hi,Yes, I will upload the source code later

Reply:
@ArjanOnwezen 
https://github.com/zhang00963/portapack-mayhem/tree/test/tools/Portapack_Hardware_version_select

Reply:
Thanks @zhang00963  I'll try to look at this this weekend to test if it works with the H1 as well without overwriting other screens / persistent memory.